core/src/main/java/com/javadiscord/jdi/core/Discord.java

```diff
@@ -152,6 +155,29 @@ public void start() {
         EXECUTOR.execute(discordRequestDispatcher);
     }
 
+    public void stop() {
+        if (this.webSocketManager != null) {
+            this.webSocketManager.stop();
+        }
+
+        discordRequestDispatcher.stop();
+
+        EXECUTOR.shutdown();
+        try {
+            if (!EXECUTOR.awaitTermination(30, TimeUnit.SECONDS)) {
+                EXECUTOR.shutdownNow();
+                if (!EXECUTOR.awaitTermination(30, TimeUnit.SECONDS)) {
+                    LOGGER.warn(
+                            "Executor failed to shutdown within the specified time limit, some"
```

Comment:
Can you advice on why the message is a concat of 2 strings?

Reply:
that is probably just spotless

Reply:
That needed to be formatted so that spotless is green, else its complaining since its over the ruler, therefore too long :-/